pkg/utils/kubernetes/crd.go

```diff
@@ -75,3 +82,35 @@ func WaitUntilCRDManifestsDestroyed(ctx context.Context, c client.Client, crdNam
 	}
 	return flow.Parallel(fns...)(ctx)
 }
+
+var (
+	crdScheme  *runtime.Scheme
+	crdCodec   runtime.Codec
+	initOnceFn sync.Once
+)
+
+func initializeCRDSchemeAndCodec() {
```

Comment:
Why is this not an init func?

Reply:
if you wish to have an init func then i can change this. Effect of calling this function from within sync.Once is the same. The initialization will just happen once.

Reply:
Yes, let's please use an init function, at least this is the style in other similar places.

Reply:
addressed in https://github.com/gardener/gardener/pull/11545/commits/e469199967b1256227f4367254263ca7d2db08db and https://github.com/gardener/gardener/pull/11545/commits/6dc30ab5771f21b85f76dbc8f54943128ad196d3